### DISCUSSION TITLE
Markdown reporter

### DISCUSSION THREAD

**Discussion by TylerGrantSmith** [ropensci/targets#693] [Help] [answered]
I like to run script-based pipelines with the summary reporter.  Is there a way to do that when knitting a pipeline in a markdown file?  Something that can be watched as the file processes.

**Comment by wlandau** [comment, marked as answer]
I recommend having a look at the functions at https://docs.ropensci.org/targets/reference/index.html#section-progress. These utilities let you monitor the progress of a pipeline in a separate R session, and many of them are more informative than the `tar_make()` reporters.
